cmake/modules/VersionInfo.cmake

```diff
@@ -7,7 +7,8 @@ IF(GIT_FOUND AND NOT FORCE_VERSION)
 		WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}"
 		TIMEOUT 1
 		OUTPUT_STRIP_TRAILING_WHITESPACE)
-	STRING(REPLACE "-" ";" TAG_LIST "${GIT_TAG}")
+	STRING(REPLACE "-" ";" GIT_TAG "${GIT_TAG}")
+	STRING(REPLACE "g" "" TAG_LIST "${GIT_TAG}")
```

Comment:
This would be better placed in the `STRING(REPLACE "v" "" FORCE_VERSION "${FORCE_VERSION}")` block incase a build ever has a `g` in it.

Furthermore, the old logic could be improved I think.  If we ever change to semantic style `1.2.0-RC2`, the `-` count should be changed to `NOT (TAG_LIST_LENGTH LESS 2)` (since cmake doesn't support `>=`).